src/Common/tests/System/Net/Http/Http2LoopbackConnection.cs

```diff
@@ -522,6 +530,7 @@ public async Task<(int streamId, HttpRequestData requestData)> ReadAndParseReque
             Assert.Equal(FrameFlags.EndHeaders, FrameFlags.EndHeaders & headersFrame.Flags);
 
             int streamId = headersFrame.StreamId;
+            requestData.Handle = (IntPtr)streamId;
```

Comment:
should the handle be just named StreamId and be `int`/`int?`?

Reply:
I could but purposely I did not want to make it HTTP/2 specific.

Reply:
fair enough, at least some comment above the handle should be sufficient

Reply:
I'm sympathetic to the idea of this not being HTTP2 specific, but I'm not sure that making it an IntPtr actually helps anything here.

I would suggest making it an int and calling it RequestId. "Handle" is extremely vague. "RequestId" at least conveys something about the meaning intended here.

We will never use this for HTTP/1.1, so it only really matters if we need it for something like HTTP/3, and we'll need to revisit then anyway.

Reply:
yes. I like that idea @geoffkizer.